lib/cmd/fh3/notifications/list.js

```diff
@@ -0,0 +1,50 @@
+/* globals i18n */
+var common = require('../../../common.js');
+var fhreq = require("../../../utils/request");
+var util = require('util');
+
+module.exports = {
+  'desc' : i18n._('List notifications of a RHMAP App'),
+  'examples' : [{
+    cmd : 'fhc notifications list --app=<app> --env=<environment> --audit=<true>',
+    desc : i18n._('List notifications of the <app>')}],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app':'a',
+    'env':'e',
+    'audit':'au',
+    'json': 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'audit',
```

Comment:
The idea in this moment is just moving to the format V3, and use `yargs` and add the unit test. Improvements in the `yargs` options and parameters can be done after for all commands in the same time if then follow the same standard and no just in this one. 

I understand that your concern here is about the json parameter which has already a JIRA planned for at in the end of all we check how we can improve that. Also, for we are able to use more implementations that yargs can offer we need to update the version of it in the project. ( which worked fine in my tests either, however, I think it should not be done in this task )

Added your comment into : [FH-3650](https://issues.jboss.org/browse/FH-3650). 

👍 Tks for the review and contribution